src/main/java/com/auction/retailService/entity/StoreProductBean.java

```diff
@@ -0,0 +1,32 @@
+package com.auction.retailService.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class StoreProductBean {
```

Comment:
Some Java objects have *Bean, some *DTO some nothing.  Nonetheless the readme does not explain why did you choose that naming convention. What is the difference in this solution between Bean and Dto?